client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
+func (client *Client) Package(pkgID string) (Package, error) {
```

Comment:
I've pinged you on Telegram re: pass-by-reference, cheers.

The argument here is the qualified package name, so in keeping the terminology consistent I think you're right in that `id` isn't used elsewhere from what I can see.